### PR TITLE
add FLT announcement

### PR DESCRIPTION
**Pull request by kbuzzard**
I've written a blog post about the FLT project, which I plan to turn into a public github repo on Tuesday 30th April.

### CODE REVIEW

File: posts/FLT-announcement.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Kevin Buzzard discusses the project to prove Fermat's Last Theorem in Lean.
+
+<!-- TEASER_END -->
+# Introduction
+
+Fermat's Last Theorem (FLT) is the claim that some abstract equation has no solutions in positive integers.
+The result has no practical applications.
+So why did the announcement of its resolution by Andrew Wiles in 1993 make the front page of the New York Times?
+
+One aspect of FLT is that the theorem is very simple to *state* (`x^n+y^n=z^n` has no solutions in positive integers if `n>=3`), and yet it was incredibly hard to *prove* (indeed, it took the mathematical community over 350 years).
```
Comment (eric-wieser) [inline posts/FLT-announcement.md:23]: nit: I think you can use `$` math here, unless the use of lean-like code is deliberate!

File: posts/FLT-announcement.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+The fact that the area is still active is part of my motivation to formalise a proof of FLT.
+
+
+# Formalisation of mathematics
+
+Formalisation of mathematics is the art of translating paper mathematics into a computer programming language rich enough to understand the concepts of theorems and proofs.
+These programming languages, also called interactive theorem provers (ITPs), have been around for decades.
+Over the last few years however, the area seems to have captured the attention of parts of the mathematics community.
+We have now seen several examples of fashionable modern research mathematics being formalised in real time, the most recent of which (at the time of writing) is Tao et al's formalisation of his work with Gowers, Green and Manners resolving the polynomial Freiman--Ruzsa conjecture.
+This breakthrough 2023 paper was formalised in Lean in three weeks flat.
```
Comment (eric-wieser) [inline posts/FLT-announcement.md:40]: Do we have a canonical link for this? Tao's blog perhaps, though I don't see a "we finished it" post.

**Comment by pitmonticone** [comment]
Just opened a simple [PR](https://github.com/kbuzzard/blog/pull/1) fixing a typo.

**Comment by eric-wieser** [comment]
@pitmonticone, I think it would be better to just leave that as a suggested change on this PR?

(I pushed a commit to put sentences on their own line to make that approach more viable)

**Comment by pitmonticone** [comment]
> @pitmonticone, I think it would be better to just leave that as a suggested change on this PR?
> 
> (I pushed a commit to put sentences on their own line to make that approach more viable)

Ok great. Thanks.

File: posts/FLT-announcement.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+However this is where one begins to see the benefits of a formalisation project.
+I will be able to explicitly *state* the results I need in Lean, and pass them on to others.
+The beauty of the system: you do not have to understand the whole proof of FLT in order to contribute.
+The blueprint breaks down the proof into many many small lemmas, and if you can formalise a proof of just one of those lemmas then I am eagerly awaiting your pull request.
+
+
+The project will be run on the [FLT stream](https://leanprover.zulipchat.com/#narrow/stream/416277-FLT) on the [Lean Zulip chat](https://leanprover.zulipchat.com/), a high-powered research forum where mathematicians and computer scientists can collaborate in real time, effortlessly posting code and mathematics using a thread and stream system which admirably handles the task of enabling many independent conversaions to happen simultaneously.
```
Comment (pitmonticone) [inline posts/FLT-announcement.md:75]: There is a typo in 

> [...] independent ~~conversaions~~ conversations [...]

**Review by pitmonticone — CHANGES_REQUESTED** [review]

File: posts/FLT-announcement.md (hunk 3 above)
Comment (eric-wieser) [reply posts/FLT-announcement.md:75]:
```suggestion
The project will be run on the [FLT stream](https://leanprover.zulipchat.com/#narrow/stream/416277-FLT) on the [Lean Zulip chat](https://leanprover.zulipchat.com/), a high-powered research forum where mathematicians and computer scientists can collaborate in real time, effortlessly posting code and mathematics using a thread and stream system which admirably handles the task of enabling many independent conversations to happen simultaneously.
```

File: posts/FLT-announcement.md (hunk 2 above)
Comment (pitmonticone) [reply posts/FLT-announcement.md:40]: The [blueprint](https://teorth.github.io/pfr/blueprint/index.html)? The [dependency graph](https://teorth.github.io/pfr/blueprint/dep_graph_document.html)? The Quanta Magazine [article](https://www.quantamagazine.org/a-team-of-math-proves-a-critical-link-between-addition-and-sets-20231206/)?

Comment (eric-wieser) [reply posts/FLT-announcement.md:40]: The last one sounds good to me

File: posts/FLT-announcement.md (hunk 1 above)
Comment (madvorak) [reply posts/FLT-announcement.md:23]: I prefer the current version.

Comment (eric-wieser) [reply posts/FLT-announcement.md:23]: I also don't know how to preview, but I think that's fine; it can be fixed in post if necessary.

Comment (kbuzzard) [reply posts/FLT-announcement.md]: OK I've put it in maths mode and changed the false to true, we'll see what happens.

**Review by pitmonticone — APPROVED** [review]

**Comment by PatrickMassot** [comment]
@joelriou bors is not watching this repo.

**Comment by kbuzzard** [comment]
OK so I will merge this in a couple of hours. Thanks for the comments!

**Pull request merged**